modules/tasks/tasks/src/main/resources/webapp/js/controllers.js

```diff
@@ -154,7 +154,7 @@
                 $('.setHideActive').find('i').removeClass("fa-square-o").addClass('fa-check-square-o');
             } else {
                 $scope.hideActive = true;
-                $scope.setFilter($scope.hidePaused ? 'noItems' : 'activeTaskFilter');
+                $scope.setFilter($scope.hidePaused ? 'allItems' : 'activeTaskFilter');
```

Comment:
remove this changes in task controller. This ticket regards API endpoints, so you should not make changes in frontend